change/@microsoft-teams-js-b8262513-d58f-4a86-9784-c732d9be6fc9.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Changed `sendAndHandleSdkError` to `sendAndUnwrap` in call.ts",
```

Comment:
It would be good to phrase what this fix is doing in a way that will make sense to external developers / consumers of teamsjs. Maybe something like "`call.startCall` no longer throws after executing successfully".